File: src/main/java/threads/ThreadPictureDefaultBehaviour.java
package threads;

import java.util.List;

import modules.picture.PictureModule;
import net.dv8tion.jda.api.events.message.MessageReceivedEvent;

public class ThreadPictureDefaultBehaviour implements Runnable {
    
    MessageReceivedEvent event;
    List<String> args;

    public ThreadPictureDefaultBehaviour(MessageReceivedEvent event, List<String> args) {
	this.event = event;
	this.args = args;
    }

    @Override
    public void run() {
	PictureModule.getByDefault(event, args);
    }

}

File: src/main/kotlin/threads/ThreadPictureDefaultBehaviour.kt
package threads

import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class ThreadPictureDefaultBehaviour(internal var event: MessageReceivedEvent, internal var args: List<String>) : Runnable {

    override fun run() {
//        PictureModule.getByDefault(event, args)
    }

}
